unblob/handlers/__init__.py

```diff
@@ -44,3 +44,7 @@
         xz.XZHandler,
     ),
 ]
+
+ALL_HANDLER_CLASSES = [
```

Comment:
The name of this variable is inconsistent with the variable it is using. You can simply name it `ALL_HANDLERS`